src/components/Images.tsx

```diff
@@ -26,7 +26,7 @@ export default function Images({
     <img
       className={className}
       key={index}
-      src={assetUrl(image.file)}
+      src={connection.getAssetUrl?.(image.file) ?? assetUrl(image.file)}
```

Comment:
Why would this be needed? Wouldn't you always have v2.1.0 of the client lib so this method will always exist?

Reply:
I think it's possible for your package-lock.json or yarn.lock to point to `cogs-client@2.0.0` with `cogs-client-react@2.1.0`.

The alternative would be to set the `cogs-client` dependency in this repo to be `>=2.1.0`. What do you think?

Reply:
I think `cogs-client@~2.1` would also work?

Reply:
I've pushed an update that uses `cogs-client@~2.1` and removes this fallback logic.

Reply:
I think that's correct 👍